.github/ISSUE_TEMPLATE.md

```diff
@@ -27,6 +27,16 @@ Need Support?
 
 ## Context (Environment)
 
+### Output of `iotedge check --output json`
+
+<details>
+<summary>Click here</summary>
```

Comment:
The `<summary>` blocks for logs (farther down) have the title inside the summary tags, but this one has "Output of `iotedge check --output json`" in a heading, and then "Click here" in the summary tags. Should the title be inside the summary tags (so, no "Click here"), rather than a header?

Reply:
Yeah, I couldn't think of what to use for the title if I made `iotedge check --output json` the summary. You're suggesting using the same thing for both?

Reply:
I guess I'm suggesting:
<code>
<details>
<summary>Output of `iotedge check --output json`</summary>

```
<paste here>
```
</details>
</code>

Reply:
I see what you're saying now. Yeah it should have a heading to be consistent with the other items under "Context". So keep it like it is.